theme/brb/views/authentication/home.hbs

```diff
@@ -20,16 +20,13 @@
 						</div>
 					{{/hasConfig}}
                     <h4>Herzlich willkommen bei der Schul-Cloud Brandenburg!</h4>
-                    <p>„Das Lernen mit digitalen Medien ist die Herausforderung mit der größten Dynamik. Dazu braucht es eine leistungsfähige, digitale Infrastruktur. Zu den wichtigsten Vorhaben gehört die am Hasso-Plattner-Institut entwickelte Schul-Cloud Brandenburg. Sie soll die Schulen des Landes vernetzen, so dass Lehrkräfte, Schülerinnen und Schüler jederzeit und überall Zugang zu Lern- und Lehrmaterialien haben. Die professionelle zentrale Wartung für alle Schulen verringert deutlich den Verwaltungs- und Betreuungsaufwand. Wir versprechen uns von der Schul-Cloud Brandenburg einen deutlichen Schub in der digitalen Bildung und freuen uns auf den Start des Projekts.“ - Bildungsministerin Britta Ernst bei der Eröffnungsveranstaltung am 2. Mai 2019.</p>
-					<p>Das Land Brandenburg pilotiert die vom Hasso-Plattner-Institut entwickelte Schul-Cloud Brandenburg mit zunächst 51 Schulen. Hinzu kommen drei Schulen aus dem HPI/MINT-EC-Projekt. Die Pilotierung erfolgt in den Schuljahren 2019/2020 und 2020/2021.</p>
-					<h4>Sie setzt besondere Schwerpunkte bei der</h4>
-					<ul>
-						<li>Verzahnung der Schul-Cloud Brandenburg mit dem im MBJS entwickelten Online-Schulverwaltungsprogramms weBBschule,</li>
-						<li>Bezugnahme auf den in Brandenburg gültigen Rahmenlehrplan 1-10 (einschl. der Nutzung von bereits existierenden offenen Schnittstellen zum RLP online),</li>
-						<li>systematischen Einbeziehung von freien Bildungsmedien (Open Educational Resources (OER)) und</li>
-						<li>Einbindung von Fortbildungsmodulen (Blended-Learning).</li>
-					</ul>
-					<p>Grundsätzliche Alleinstellungsmerkmale der HPI-Entwicklung sind ihre Prozessorientierung, Skalierbarkeit und damit zukunftsgerichtete Nachhaltigkeit.</p>
+                    <p>"Das Lernen mit digitalen Medien ist die Herausforderung mit der größten Dynamik. Dazu braucht es eine leistungsfähige, digitale Infrastruktur. Zu den wichtigsten Vorhaben gehört die am Hasso-Plattner-Institut (HPI) entwickelte Schul-Cloud Brandenburg. Sie soll die Schulen des Landes vernetzen, so dass Lehrkräfte, Schülerinnen und Schüler jederzeit und überall Zugang zu Lern- und Lehrmaterialien haben. Die professionelle zentrale Wartung für alle Schulen verringert deutlich den Verwaltungs- und Betreuungsaufwand. Wir versprechen uns von der Schul-Cloud Brandenburg einen deutlichen Schub in der digitalen Bildung und freuen uns auf den Start des Projekts.“ - Bildungsministerin Britta Ernst bei der Eröffnungsveranstaltung am 2. Mai 2019.</p>
```

Comment:
Both options are valid, but at least the Duden recommends "sodass" (https://www.duden.de/rechtschreibung/sodass)

```suggestion
                    <p>"Das Lernen mit digitalen Medien ist die Herausforderung mit der größten Dynamik. Dazu braucht es eine leistungsfähige, digitale Infrastruktur. Zu den wichtigsten Vorhaben gehört die am Hasso-Plattner-Institut (HPI) entwickelte Schul-Cloud Brandenburg. Sie soll die Schulen des Landes vernetzen, sodass Lehrkräfte, Schülerinnen und Schüler jederzeit und überall Zugang zu Lern- und Lehrmaterialien haben. Die professionelle zentrale Wartung für alle Schulen verringert deutlich den Verwaltungs- und Betreuungsaufwand. Wir versprechen uns von der Schul-Cloud Brandenburg einen deutlichen Schub in der digitalen Bildung und freuen uns auf den Start des Projekts.“ - Bildungsministerin Britta Ernst bei der Eröffnungsveranstaltung am 2. Mai 2019.</p>
```